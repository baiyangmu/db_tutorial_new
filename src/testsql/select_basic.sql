create table users (id int, username string, email string)
use users
insert into users 1 alice alice@example.com
insert into users 2 bob bob@example.com
select * from users
select id, username from users where id = 1
.exit
