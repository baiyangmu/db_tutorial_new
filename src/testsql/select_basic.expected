Table 'users' created with 3 columns.
Executed.
Using table 'users'.
Executed.
Executed.
(1, alice, alice@example.com)
(2, bob, bob@example.com)
(1, alice)
Executed.
